flat-manager-client

```diff
@@ -581,7 +581,7 @@ async def publish_build(session, build_url, wait, token):
                 elif current_state == "validating":
                     print("the build is still being validated or held for review")
                     return {}
-            except:
```

Comment:
This bare except in particular would silently swallow the ApiError explicitly raised within the above try block.